### PR TITLE
Added my blog URL to my guest posts

### PR DESCRIPTION
**Pull request by discreet**

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
@discreet you have the same url in both posts, dont you want `https://chrispisano.wordpress.com/2016/01/18/journey-to-high-availability-2/` and whatever the url of the other post ends up being when you post it?

File: _posts/2015-12-03-journey_to_high_availability.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ tags:
 - puppet
 - high_availability
 guest_post: true
+guest_url:
+https://chrispisano.wordpress.com/2016/01/18/journey-to-high-availability-2/
```
Comment (GregSutcliffe) [inline _posts/2015-12-03-journey_to_high_availability.md]: this should be a single line, but I can fix that on merge if you like ;)

Comment (discreet) [inline _posts/2015-12-03-journey_to_high_availability.md]: ah, must be my 80 character/line limit for .md files in vim. . . i can fix it today

**Comment by dLobatog** [comment]
Merged as 668b7bb98ab9d946a2508a32c129efe5aaf3948e, thanks @discreet!

**Pull request closed without merging**